ci-operator/config/openshift-assisted/assisted-grafana/openshift-assisted-assisted-grafana-master.yaml

```
@@ -1,8 +1,17 @@
+base_images:
+  coreos_centos_stream8:
+    name: stream8
+    namespace: coreos
+    tag: "8"
 build_root:
-  image_stream_tag:
-    name: release
-    namespace: openshift
-    tag: golang-1.18
+  project_image:
+    dockerfile_literal: |
+      FROM registry.ci.openshift.org/origin/centos:stream8
+      RUN dnf install git -y
+images:
+- dockerfile_path: Dockerfile.build
+  from: src
+  to: build
```

iiuc currently ``coreos_centos_stream8`` is unused and will get removed automatically with a certain automatic action
To use it (and leverage prow's internal registry) we need to do:
```suggestion
- dockerfile_path: Dockerfile.build
  from: src
  inputs:
    coreos_centos_stream8:
      as:
      - quay.io/centos/centos:stream8
  to: build
```